src/DB/QueryBuilder/Concerns/CRUD.php

```diff
@@ -48,6 +48,28 @@ public function update( $data, $format = null ) {
 		);
 	}
 
+	/**
```

Comment:
I know not every method here has a nice method/param/return description, but I think it definitely makes sense to add it, including param types.